content/applications/general/email_communication/faq.rst

```diff
@@ -1,174 +1,205 @@
    :align: center
    :alt: Email scheduled to be sent later.
 
-By default, the Mass Mailing cron runs every 60 minutes. So, you should wait maximum an hour
-before the campaign is actually sent.
+Emails that are considered urgent (communication from one person to another, such as: sales orders,
+invoices, purchase orders, etc.) are sent directly (immediately).
 
 Incoming emails
 ===============
 
-When you have an issue with incoming emails, there might not be an indication per se in Odoo.
-This is the client who tries to contact a database who will get a bounce (most of the
-time 550: mailbox unavailable).
+When there is an issue with incoming emails, there might not be an indication, per se, in Odoo. It
+is the sending email client, who tries to contact a database, that will get a bounce-back (most of
+the time a :guilabel:`550: mailbox unavailable`).
 
 Emails are not received
 -----------------------
 
-Depending on the platform you are using:
+The following steps are all dependent on the Odoo platform the database is using:
+
+- **Odoo.sh** users can find their live logs on the folder :file:`~/logs/`.
 
-- The **Odoo.sh** users can find their live logs on the folder :file:`~/logs/`.
+.. tip::
+   The folder :file:`~/logs/` (preferably accessed by the command line) of an Odoo.sh database
+   contains a list of files containing the logs of the database. The log files are created everyday
+   at 5:00 AM (UTC).
 
-- The folder :file:`~/logs/` (preferably accessed by the command line) of an Odoo.sh contains
-  a list of files containing the logs of the database. The log files are created everyday
-  at 5:00 AM UTC. The two last days are not compressed, while the older ones are, in order
-  to gain space. The naming of the files for Today and Yesterday are :file:`odoo.log` and
-  :file:`odoo.log.1`. For the following, they are named with their dates and compressed.
-  See the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and
-  ``zgrep`` (for the compressed ones) to search through the files.
+   The two last days are not compressed, while the older ones are, in order to save space. The
+   naming of the files for today and yesterday are respectively: :file:`odoo.log` and
+   :file:`odoo.log.1`.
 
-- **Odoo Online** users won’t have access to their logs. However you can still contact
-  `Odoo Support <https://www.odoo.com/help>`_ , if you have a recurring issue
-  with the same client or domain.
+   For the following days, they are named with their dates, and then compressed. See the Odoo.sh
+   documentation about :ref:`logs <odoosh/logs>`. Use the command :command:`grep` and
+   :command:`zgrep` (for the compressed ones) to search through the files.
+
+- **Odoo Online** users won't have access to the logs. However `Odoo Support
+  <https://www.odoo.com/help>`_ can still be contacted, if there is a recurring issue with the same
+  client or domain.
 
 Get help from support
```

Comment:
This section would make more sense as an H2 heading, instead of being under "incoming emails"
```suggestion
Get help from Odoo Support
```